src/get-own-property-symbols.js

```diff
@@ -89,7 +90,7 @@
         }
       };
       defineProperty(ObjectProto, uid, descriptor);
-      return (source[uid] = defineProperty(
+      return freeze(source[uid] = defineProperty(
```

Comment:
That seems like a conflict with the spec, which generally requires everything be changeable?